src/styles/index.scss

```diff
@@ -1,3 +1,47 @@
+@import './utils/variables';
+@import './blocks/card';
+@import './blocks/header';
+
 body {
   margin: 0;
+  font-family: Roboto, Arial, sans-serif;
+  font-weight: 400;
+  font-style: normal;
+}
+
+a,
+p {
+  margin: 0;
+  padding: 0;
+}
+
+a {
+  display: inline-block;
+}
+
+.main {
+  padding: 50px 40px;
+  gap: 46px 48px;
+
+  display: grid;
+  grid-template-columns: repeat(1, 200px);
```

Comment:
According to the BEM methodology, it's not recommended to use tags for styling. Please, create a class for body tag and apply the styles to this class.